cfme/fixtures/v2v_fixtures.py

```diff
@@ -290,14 +289,25 @@ def delete_conversion_hosts(appliance):
             "Failed to delete all conversion hosts: {}".format(delete_hosts.output))
 
 
-def __configure_conversion_host_ui(appliance, target_provider, hostname, default,  # noqa
-                                   conv_host_key, transformation_method,
-                                   vmware_ssh_key, osp_cert_switch=None, osp_ca_cert=None):
-
+def configure_conversion_host_ui(appliance, target_provider, hostname, default_value,
```

Comment:
Consider:

This method doesn't really _do_ anything, other than pipe all its kwargs into the create method for conversion host. Is it really necessary?

Reply:
Earlier this method was calling helper methods from this file and filling all parameters of conversion host .
Moved all those methods to test file itself now . Kept it a method itself as that reduces some lines of code